http.h

```diff
@@ -0,0 +1,18 @@
+#include <string>
+#include <unordered_map>
+
```

Comment:
this should probably live inside of a namespace if were gonna follow the google style guide (see this section on [namespaces](https://google.github.io/styleguide/cppguide.html#Namespaces))

we could do something like

```cpp
namespace domino {
namespace http {
  enum method { GET, POST };
  struct request {
    method method;
    // if the url was http://localhost:3000/upload?param=1
    // the endpoint should just be "upload"
    std::string endpoint;
  };
} // namespace http
} // namespace domino
```

then things can get super clean like

```cpp
domino::http::method::GET;
domino::http::method::POST;

domino::http::request request;

request.method
request.endpoint;
```